llama/CMakeLists.txt

```diff
@@ -6,18 +6,17 @@ include(FetchContent)
 FetchContent_Declare(
     llama_cpp
     GIT_REPOSITORY https://github.com/ggerganov/llama.cpp.git
-    GIT_TAG        55dbb91
+    GIT_TAG        1d16309
 )
 
 FetchContent_MakeAvailable(llama_cpp)
 
-add_library(binding ${CMAKE_CURRENT_SOURCE_DIR}/binding/binding.cpp ${llama_cpp_SOURCE_DIR}/examples/common.cpp)
-target_include_directories(binding PRIVATE ${llama_cpp_SOURCE_DIR}/examples)
-target_link_libraries(binding llama ggml_static)
+file(COPY ${llama_cpp_SOURCE_DIR}/llama.h DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
We could commit these files, they're 500-700kb for each arch / os combo. The cuda ones may be too big so I kept it as an addl' build step for now